electron-quick-start-es6/.vscode/launch.json

```diff
@@ -7,13 +7,20 @@
       "request": "launch",
       "cwd": "${workspaceRoot}",
       "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/electron",
-      "program": "${workspaceRoot}/main.js"
+      "windows": {
+        "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/electron.cmd"
+      },
+      "program": "${workspaceRoot}/main.js",
+      "protocol": "legacy"
```

Comment:
`"protocol": "legacy"` seems to be only available in insider, so maybe keep this part out until next month?